File: Python/S0Fzl-03.py
def main():
    a = float(input("Please enter your grade: "))
    if a <= 20 and a > 0:
        if a >= 10:
            print("Pass")
        else:
            print("Fail")
    else:
        print("Grade out of range.")


if __name__ == "__main__":
    main()

File: C/S0Fzl-03.c
#include <stdio.h>

int main(){
        float a;
        printf("Please enter your grade: ");
        scanf ("%f", &a);
        if (a>=0 && a<=20) {
                if (a>=10)
                        printf("Pass.\n");
                else
                        printf("Fail.\n");
        } else
                printf("your grade is out of range.\n");
        return 0;
}
